dev_scripts/exportTranslation.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable default-case */
+/* eslint-disable array-callback-return */
+/* eslint-disable func-names */
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable global-require */
+/* eslint-disable no-console */
+/* eslint-disable no-useless-escape */
+/* eslint-disable no-shadow */
+/* eslint-disable guard-for-in */
+/* eslint-disable prefer-const */
+/* eslint-disable no-undef */
+/* eslint-disable no-restricted-syntax */
+
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+const fs = require('fs');
+
+const localizationFiles = [];
+let index = 0;
+
+// Define mainLanguage as English (gb)
+const mainLanguage = 'gb';
+
+getLocalizationFiles = () => {
+  // Reads all files in localization files and save them in localizationFiles array
+  fs.readdirSync('././src/localization/').forEach(file => {
+    // Filter to only get JSON files
+    if (file.split('.').pop() === 'json') {
+      const fileName = require(`../src/localization/${file}`);
+      localizationFiles.push(fileName);
+      localizationFiles[index].name = file.replace('.json', '');
+      index += 1;
+    }
+  });
+};
+
+// Gets the language entered by the User (fr, la, gil, tl)
+getSelectedLanguage = () => process.argv[2];
+
+generateCsvData = (mainLanguage, selectedLanguage) => {
+  let valueFound = false;
+  // csvFileData keeps all the data stored in json files
+  let csvData = '';
+  // Goes through all localization files
+  localizationFiles.map(localizationFile => {
+    const mainLanguageKeys = [];
+    const mainLanguageValues = [];
+    const selectedLanguageValues = [];
+    // Iterates over every language
+    for (const [language, mainLanguageKey] of Object.entries(localizationFile)) {
+      switch (language) {
+        case mainLanguage:
+          // Saves all mainLanguageValues
+          for (let [mainLanguageKeyAgain, mainLanguageValue] of Object.entries(mainLanguageKey)) {
+            mainLanguageKeys.push(mainLanguageKeyAgain);
+            mainLanguageValue = mainLanguageValue
+              .replace(/\n/gi, '{nextLine}')
+              .replace(/[\"]/gi, '{emptySpace}');
```

Comment:
I don't think this is needed (`emptySpace` one)

Reply:
Totally agree, removed! I just kept {nextLine} declared as a constant
`const nextLine = /{nextLine}/g;`